k8s/core/pods.go

```diff
@@ -297,6 +273,59 @@ func (c *Client) GetPodByUID(uid types.UID, namespace string) (*corev1.Pod, erro
 	return nil, schederrors.ErrPodsNotFound
 }
 
+// DeletePods deletes the given pods
+func (c *Client) DeletePods(pods []corev1.Pod, force bool) error {
+	for _, pod := range pods {
+		if err := c.DeletePod(pod.Name, pod.Namespace, force); err != nil {
+			return err
+		}
+	}
+
+	return nil
+}
+
+// DeletePod deletes the given pod
+func (c *Client) DeletePod(name string, ns string, force bool) error {
+	if err := c.initClient(); err != nil {
+		return err
+	}
+
+	deleteOptions := metav1.DeleteOptions{}
+	if force {
+		gracePeriodSec := int64(0)
+		deleteOptions.GracePeriodSeconds = &gracePeriodSec
+	}
+
+	return c.kubernetes.CoreV1().Pods(ns).Delete(context.TODO(), name, deleteOptions)
```

Comment:
Let's call the methods from `common` here so we don't duplicate code.

Reply:
Calling GetPods() from the common now, GetPod() is not in common